libs/designer/src/lib/ui/templates/review/ReviewCreatePanel.tsx

```diff
@@ -0,0 +1,151 @@
+      description: 'Accessibility label for workflow name',
+    }),
+    STATE_TYPE: intl.formatMessage({
+      defaultMessage: 'State type',
+      id: 'tzeDPE',
+      description: 'Accessibility label for state kind',
+    }),
+    PLACEHOLDER: intl.formatMessage({
+      defaultMessage: '----',
+      id: 'wPi8wS',
+      description: 'Accessibility label indicating that the value is not set',
+    }),
+    kind_stateful: intl.formatMessage({
+      defaultMessage: 'Stateful',
+      id: 'Qqmb+W',
+      description: 'Dropdown option for stateful type',
+    }),
+    kind_stateless: intl.formatMessage({
+      defaultMessage: 'Stateless',
+      id: 'cNXS5n',
+      description: 'Dropdown option for stateless type',
+    }),
+    CREATE_VIEW_NO_CONFIG: intl.formatMessage({
+      defaultMessage: 'Select Create to create a new workflow based on this template, no configuration required.',
+      id: 'uOU0lL',
+      description: 'Accessibility label for no configuration required',
+    }),
+    UPDATE_VIEW_NO_CONFIG: intl.formatMessage({
+      defaultMessage: 'Select Update to update this workflow based on this template, no configuration required.',
+      id: 'OaUode',
+      description: 'Accessibility label for no configuration required',
+    }),
+  };
+
+  return (
+    <div className="msla-templates-tab">
+      {!isConsumption && (
+        <>
+          <Label className="msla-templates-tab-label" htmlFor={'detailsLabel'}>
+            {intlText.BASICS}
+          </Label>
+          <div className="msla-templates-tab-review-section">
+            {Object.values(workflows).map((workflow) => {
+              const workflowNameToShow = existingWorkflowName ?? workflow.workflowName;
+              return (
+                <div key={workflow.id}>
+                  <div className="msla-templates-tab-review-section-details">
+                    <Text className="msla-templates-tab-review-section-details-title">{intlText.WORKFLOW_NAME}</Text>
+                    <Text className="msla-templates-tab-review-section-details-value">
+                      {isUndefinedOrEmptyString(workflowNameToShow) ? intlText.PLACEHOLDER : workflowNameToShow}
+                    </Text>
+                  </div>
+                  <div className="msla-templates-tab-review-section-details">
+                    <Text className="msla-templates-tab-review-section-details-title">{intlText.STATE_TYPE}</Text>
+                    <Text className="msla-templates-tab-review-section-details-value">
+                      {equals(workflow.kind, WorkflowKind.STATEFUL)
+                        ? intlText.kind_stateful
+                        : (intlText.kind_stateless ?? intlText.PLACEHOLDER)}
+                    </Text>
+                  </div>
+                </div>
+              );
+            })}
+          </div>
+        </>
+      )}
+
+      {isConsumption && !Object.keys(connections).length && !Object.keys(parameterDefinitions).length ? (
```

Comment:
Do you mind adding the section to for subscription/resourcegroup/location here as well in this format when enableresourceselection is true in the options store please?